packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -708,16 +706,15 @@ export default class NewVm extends BaseComponent {
     }
 
     const automaticNetworks = this._getAutomaticNetworks()
-    if (automaticNetworks.length !== 0) {
-      return automaticNetworks
-    }
 
     const network = find(this._getPoolNetworks(), network => {
       const pif = getObject(store.getState(), network.PIFs[0])
       return pif && pif.management
     })
 
-    return network !== undefined ? [network.id] : []
+    return network !== undefined
+      ? [network.id, ...automaticNetworks]
+      : [...automaticNetworks]
```

Comment:
I think we shouldn't keep the management network if the user has some automatic networks.